packaging/installer/README.md

```diff
@@ -202,11 +202,11 @@ dnf install zlib-devel libuuid-devel libuv-devel lz4-devel Judy-devel openssl-de
 yum install autoconf automake curl gcc git libmnl-devel libuuid-devel openssl-devel libuv-devel lz4-devel Judy-devel lm_sensors make MySQL-python nc pkgconfig python python-psycopg2 PyYAML zlib-devel
 
 # openSUSE
-zypper install zlib-devel libuuid-devel libuv-devel liblz4-devel judy-devel openssl-devel libmnl-devel gcc make git autoconf autoconf-archive autogen automake pkgconfig curl findutils
+zypper install zlib-devel libuuid-devel libuv-devel liblz4-devel judy-devel libopenssl-devel libmnl-devel gcc make git autoconf autoconf-archive autogen automake pkgconfig curl findutils
 
 ```
 
-Please note that for RHEL/CentOS you might need [EPEL](http://www.tecmint.com/how-to-enable-epel-repository-for-rhel-centos-6-5/).
+Please note that for RHEL/CentOS you might need [EPEL](http://www.tecmint.com/how-to-enable-epel-repository-for-rhel-centos-6-5/). Especially for using `memory mode = dbengine` EPEL is strictly required. In addition, RHEL/CentOS version 6 also needs [OKay](https://okay.com.mx/blog-news/rpm-repositories-for-centos-6-and-7.html) for package libuv version 1. Arch linux needs arch/aur for package Judy.
```

Comment:
Move the information inside the system-specific list:
```
Alpine Linux and its derivatives (you have to install bash yourself, before using the installer)
Arch Linux and its derivatives
Gentoo Linux and its derivatives
Debian Linux and its derivatives (including Ubuntu, Mint)
Fedora and its derivatives (including Red Hat Enterprise Linux, CentOS, Amazon Machine Image)
SuSe Linux and its derivatives (including openSuSe)
SLE12 Must have your system registered with Suse Customer Center or have the DVD. See #1162
```

We want everyone to have dbengine enabled, so it's not a SHOULD, but a MUST.